ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/MessageScrubber.java

```diff
@@ -23,13 +23,20 @@
    * Special exception to signal that a message is affected by a specific bug and should
    * be written to error output.
    */
-  public static class AffectedByBugException extends Exception {
+  public static class AffectedByBugException extends RuntimeException {
 
     public AffectedByBugException(String bugNumber) {
       super(bugNumber);
     }
   }
 
+  /**
+   * Special exception class that signals that a given message should not be sent
+   * downstream to either success or error output.
+   */
+  public static class MessageShouldBeDroppedException extends RuntimeException {
+  }
```

Comment:
I moved this out of `MapWithErrors` since `MapWithErrors` will eventually get removed after refactoring everything